docs/how-to-guides/themes/theme-json.md

```diff
@@ -877,17 +882,48 @@ p { /* The core/paragraph opts out from the default behaviour and uses p as a se
 
 {% end %}
 
+#### Referencing a style
+
+A block can be styled using a reference to a root level style. This feature is supported by Gutenberg.
+If you register a background color for the root using styles.color.background:
+
+```JSON
+"styles": {
+		"color": {
+			"background": "var(--wp--preset--color--primary)"
+		}
+	}
+```
+
+You can use `ref: "styles.color.background"`  to re-use the style for a block:
+
+```JSON
+{
+	"color": {
+		"text": { ref: "styles.color.background" }
+	}
+}
+```
+
 #### Element styles
 
-In addition to top-level and block-level styles, there's the concept of elements that can used in both places. There's a closed set of them:
+In addition to top-level and block-level styles, there's the concept of elements that can be used in both places. There's a closed set of them:
+
+Supported by Gutenberg:
+
+- `button`: maps to the `wp-element-button` and `.wp-block-button__link` CSS classes.
```

Comment:
Might be worth mentioning that `.wp-block-button__link` is there for backwards compatibility reasons.